zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to **Apps** in the bottom-right corner.
+
+1. Then, Go to **Admin Panel** -> **Webhooks**.
```

Comment:
Same, if both are clickable elements, split it up like this: `Click on **Admin Panel, and click on **Webhooks**.`